src/System.Net.Security/src/System/Net/SslStreamPal.Unix.cs

```diff
@@ -19,6 +19,11 @@ public static Exception GetException(SecurityStatusPal status)
             return new Interop.OpenSsl.SslException((int)status);
         }
 
+        public static bool StartMutualAuthAsAnonymous
+        {
+            get { return false; }
+        }
```

Comment:
Can we just make this a constant rather than a property?  Or if it must not be a constant for some reason, a readonly static field?